File: crates/target_csharp_system_text/output/geojson/GeojsonObjectMultiPolygon.cs
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace JtdCodegenE2E
{
    /// <summary>
    /// For type "MultiPolygon", the "coordinates" member is an array of
    /// Polygon coordinate arrays.
    /// </summary>
    public class GeojsonObjectMultiPolygon : GeojsonObject
    {
        [JsonPropertyName("type")]
        public string Type_ { get => "MultiPolygon"; }

        [JsonPropertyName("coordinates")]
        public IList<LinearRing> Coordinates { get; set; }

        [JsonPropertyName("bbox")]
        [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingDefault)]
        public BoundingBox? Bbox { get; set; }
    }
}

File: crates/target_java_jackson/output/geojson/GeojsonObjectMultiPolygon.java
package com.example;


import com.fasterxml.jackson.annotation.JsonIgnoreProperties;

import com.fasterxml.jackson.annotation.JsonInclude;

import com.fasterxml.jackson.annotation.JsonProperty;

import com.fasterxml.jackson.databind.annotation.JsonSerialize;

import java.util.List;

/**
 * For type "MultiPolygon", the "coordinates" member is an array of
 * Polygon coordinate arrays.
 */

@JsonSerialize
@JsonIgnoreProperties(ignoreUnknown = true)
public class GeojsonObjectMultiPolygon extends GeojsonObject {

    
    @JsonProperty("coordinates")
    private List<LinearRing> coordinates;

    @JsonInclude(JsonInclude.Include.NON_NULL)
    @JsonProperty("bbox")
    private BoundingBox bbox;


    public GeojsonObjectMultiPolygon() {
    }


    public List<LinearRing> getCoordinates() {
        return this.coordinates;
    }

    public void setCoordinates(List<LinearRing> coordinates) {
        this.coordinates = coordinates;
    }

    public BoundingBox getBbox() {
        return this.bbox;
    }

    public void setBbox(BoundingBox bbox) {
        this.bbox = bbox;
    }

}
